### ISSUE TITLE
Bug - sev-2 - ENG: Update Rx App Headings

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#7837]
The following updates need to be made in the Rx area of the app. 
_This list has been pulled in from the [full list of audit results](https://docs.google.com/spreadsheets/d/1KnR9Isdw8e3aBp5WhOFk7vaR4p6v4-wpYIO4GsEk4Go/edit?usp=sharing)._

- [ ] **Content:** All prescriptions ( # )..., Pending refills ( # ), etc.
     - **Location:** Health / Prescriptions
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [ ] **Content:** [ prescription name ]
     - **Location:** Health / Prescriptions
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [ ] **Content:** This list may not include all your medications
     - **Location:** Health / Prescriptions / Prescriptions help
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [ ] **Content:** [ status name ]
     - **Location:** Health / Prescriptions / Status definition
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [ ] **Content:** [ prescription name ]
     - **Location:** Health / Prescriptions / Details
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [ ] **Content:** Instructions
     - **Location:** Health / Prescriptions / Details
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [x] **Content:** Refills left
     - **Location:** Health / Prescriptions / Details
     - **Currently a heading?:** No
     - **Recommended change:** We should consider updating this to a heading in the future. However, currently it is grouped together with the additional content, so it is not easy to separate the two elements. We will leave it as is right now, but should revisit in the future.
- [x] **Content:** Fill date
     - **Location:** Health / Prescriptions / Details
     - **Currently a heading?:** No
     - **Recommended change:** We should consider updating this to a heading in the future. However, currently it is grouped together with the additional content, so it is not easy to separate the two elements. We will leave it as is right now, but should revisit in the future.
- [x] **Content:** Quantity
     - **Location:** Health / Prescriptions / Details
     - **Currently a heading?:** No
     - **Recommended change:** We should consider updating this to a heading in the future. However, currently it is grouped together with the additional content, so it is not easy to separate the two elements. We will leave it as is right now, but should revisit in the future.
- [x] **Content:** Expires on
     - **Location:** Health / Prescriptions / Details
     - **Currently a heading?:** No
     - **Recommended change:** We should consider updating this to a heading in the future. However, currently it is grouped together with the additional content, so it is not easy to separate the two elements. We will leave it as is right now, but should revisit in the future.
- [x] **Content:** Ordered on
     - **Location:** Health / Prescriptions / Details
     - **Currently a heading?:** No
     - **Recommended change:** We should consider updating this to a heading in the future. However, currently it is grouped together with the additional content, so it is not easy to separate the two elements. We will leave it as is right now, but should revisit in the future.
- [ ] **Content:** VA facility
     - **Location:** Health / Prescriptions / Details
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [ ] **Content:** [ prescription name ]
     - **Location:** Health / Prescriptions / Tracking / [prescription]
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [ ] **Content:** Package # of #
     - **Location:** Health / Prescriptions / Tracking / [prescription]
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [ ] **Content:** Tracking number
     - **Location:** Health / Prescriptions / Tracking / [prescription]
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [ ] **Content:** Prescriptions for refill ( # )
     - **Location:** Health / Prescriptions / Refill request
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [ ] **Content:** [ prescription name ]
     - **Location:** Health / Prescriptions / Refill request
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [ ] **Content:** [ prescription name ]
     - **Location:** Health / Prescriptions / Refill request / [ summary screen ]
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.
- [ ] **Content:** What's next
     - **Location:** Health / Prescriptions / Refill request / [ summary screen ]
     - **Currently a heading?:** No
     - **Recommended change:** Update this to a heading.

**Comment by wavelaurenrussell** [comment]
Additional UX recs are available for this screen.

**Comment by brea11y** [comment]
Ticket has been reviewed and is still valid.

**Comment by alexandec** [comment]
@brea11y On the Prescription Details screen, the fields that are grouped and displayed two per row are treated differently. Those are Refills left, Fill date, Expires on, and Ordered on. For those fields, the bold text and regular text are read together, like "Refills left: 5" and the whole chunk gets focus. I'm planning to leave those as is. For the fields that take up a full row, the heading gets focus separately so I'll add the role for those headings.

Similarly for the refill request items, each individual item gets focus as a block and is read as a single announcement, so I'm planning to leave those as is. Same thing for the refill request summary items.

Let me know if any of this is off.

**Comment by brea11y** [comment]
That sounds good to me. In the future, we could consider breaking out the headings and copy into separate swipes and assign the heading tag (I would recommend this). For now, though, I agree that we should leave them as-is and circle back to more complex changes at a later date.

**Comment by alexandec** [comment]
@brea11y agreed, we can split them out in the future in a different ticket.

I'm increasing this ticket to 3 points because it touches a bunch of unit tests that need improvements/additions. I'll make those changes along with the header changes.

**Comment by TKDickson** [comment]
Assigning to @DJUltraTom for QA

**Comment by DJUltraTom** [comment]
Created [This test run]( https://dsvavsp.testrail.io/index.php?/cases/view/103495) for when this is ready to be validated (by @brea11y or @DJUltraTom

**Comment by DJUltraTom** [comment]
@brea11y can you review the AC in the ticket description and revise any "Recommended change: We might want to consider making this a heading." with what the decision ended up being?

**Comment by brea11y** [comment]
Hey @DJUltraTom – I updated the list based on Chris' comment above (outlining which items he would be able to update to a heading and which items he would not). With audits, I typically make a recommendation and leave these decisions up to the teams, so I am not always aware of what the final decision ends up being. Fortunately, in this case, Chris was super thorough, so updating the list was something I could do. 🎉

We should be all set with this one. Please let me know if there is anything else that I can help with.

**Comment by DJUltraTom** [comment]
going off the basis of the comments it appears all headers have been implemented as designed. 
I revised the test run results accordingly.

PR has been approved by QA

**Comment by DJUltraTom** [comment]
Verified changes are present in overnight build. Closing ticket as deployed.
